pkg/clustermanager/reconcile_logicalcluster.go

```diff
@@ -22,9 +22,23 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	lcv1alpha1 "github.com/kubestellar/kubestellar/pkg/apis/logicalcluster/v1alpha1"
+	lcv1alpha1apis "github.com/kubestellar/kubestellar/pkg/apis/logicalcluster/v1alpha1"
 	pclient "github.com/kubestellar/kubestellar/pkg/clustermanager/providerclient"
 )
 
+const finalizerName = "LCFinalizer"
+
+// containsFinalizer: returns true if the finalizer list contains the logical cluster finalizer
+func containsFinalizer(lcluster *lcv1alpha1apis.LogicalCluster, finalizer string) bool {
```

Comment:
why don't you define it on the file that use it ?

Reply:
Not sure I understand.  finalizerName is used primarily in reconcile_logicalcluster.go.  But looking at this code I realized the code in the provider watch was going through the finalizer list twice.  I changed it to just check whether the cluster was managed.